Documentation/benchmarks/bench.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
```

Comment:
we:
1.  separate this from the doc pr
2.  move this into its own dir
3.  have a readme about the script
4.  add etcd binary setup configuration into the script